content/doc/tutorials/tutorial-for-installing-jenkins-on-AWS.adoc

```diff
@@ -31,6 +31,7 @@ If you already have one, you can skip to step 3.
 
 . An *AWS account*. If you don't have one, you can register link:https://portal.aws.amazon.com/billing/signup#/start[here].
 . An Amazon EC2 key pair. If you don't have one, refer to <<Creating a key pair>>.
+. An AWS IAM User with programmatic key access and permissions to launch EC2 instances
```

Comment:
```suggestion
. An AWS IAM User with programmatic key access and link:https://plugins.jenkins.io/ec2/#plugin-content-iam-setup[permissions to launch EC2 instances]
```

What do you think about linking to the documentation chapter which lists the required IAM permissions (as it's not only `ec2:CreateVM` which is required)?